dataverse-persistence/src/test/resources/dbinit.sql

```diff
@@ -233,6 +233,24 @@ INSERT INTO datafile (id, checksumtype, checksumvalue, contenttype, filesize, in
 INSERT INTO filetermsofuse (id, allrightsreserved, restrictcustomtext, restricttype, license_id) VALUES (120, false, NULL, 'NOT_FOR_REDISTRIBUTION', null);
 INSERT INTO filemetadata (id, description, directorylabel, label, prov_freeform, restricted, version, datafile_id, datasetversion_id, displayorder, termsofuse_id) VALUES (113, '', NULL, 'restricted.zip', NULL, NULL, 5, 58, 36, 2, 120);
 
+--- Dataset after embargo for reindex
+
+INSERT INTO dvobject (id, dtype, owner_id, previewimageavailable, protocol, authority, identifier, globalidcreatetime, identifierregistered, storageidentifier, releaseuser_id, publicationdate,
+                      creator_id, createdate, modificationtime, permissionmodificationtime, indextime)
+    VALUES (101, 'Dataset', 51, false, NULL, NULL, NULL, NULL, true, '', NULL, NULL,
```

Comment:
I hope that these new dataset won't break any existing integration tests.
It would be better to place that lines in the section for datasets (now it's in datafiles).